packages/react-router-dom/index.tsx

```diff
@@ -261,7 +261,15 @@ export interface LinkProps
  */
 export const Link = React.forwardRef<HTMLAnchorElement, LinkProps>(
   function LinkWithRef(
-    { onClick, reloadDocument, replace = false, state, target, to, ...rest },
+    {
+      onClick,
+      reloadDocument,
+      replace = undefined,
```

Comment:
I think we can drop the default value assignment here since it'll already be `undefined` without it?

```suggestion
      replace,
```